lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,30 @@
 import React from 'react';
 
+import PropTypes from 'prop-types';
+
+
 const Card = props => {
+  console.log("props in Card, ", props)
```

Comment:
Don't forget to clean up your console logs before merging your code into production.